edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/Microsoft.Azure.Devices.Edge.Agent.Diagnostics.csproj

```diff
@@ -9,8 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.33.4-NestedEdge" />
```

Comment:
nice to see that we are getting rid of custom package. Does this mean we can get rid of `azure-iotedge-packages` feed in Nuget.config? Also, if that's the case, we should also clean up all .yaml files where we suppressed `NugetSecurityAnalysisWarningLevel`. I'm fine if this could be done in a separate PR, we just need a work item to track.

Reply:
removed the nuget source